sqlglot/parser.py

```diff
@@ -236,6 +236,7 @@ class Parser(metaclass=_Parser):
         TokenType.SCHEMA,
         TokenType.TABLE,
         TokenType.VIEW,
+        TokenType.MODEL,
```

Comment:
does model need to be added to id_var?

Reply:
it's included indirectly because every `CREATABLE` is a valid identifier